src/relay/blockchain/delegate.py

```diff
@@ -112,6 +111,7 @@ def validate_meta_transaction_fees(self, meta_transaction: MetaTransaction):
                 == meta_transaction.currency_network_of_fees
                 and fees_estimation.base_fee <= meta_transaction.base_fee
                 and fees_estimation.gas_price <= meta_transaction.gas_price
+                and fees_estimation.fee_recipient == meta_transaction.fee_recipient
```

Comment:
How does that work, fee recipient does not have to be set in the config, no?
And fee recipient 0x00000 should be also fine, no?

Reply:
If it's not set in the config, it is using the node's address, isn't it?
https://github.com/trustlines-protocol/relay/blob/master/src/relay/relay.py#L338

We could also accept 0x000.